Bust Config.rb

```diff
@@ -166,6 +198,13 @@ def custom_bust_display_options
         nil,
         face_name = 'Vera emo'
       ]
+    elsif Busty::TOPLESS_SIMON_MAP[$game_message.face_name] && $game_actors[2].character_index == 2
+      # Simon is using his almost naked sprite, override with topless faceset
+      return [
+        -40,
+        19,
+        face_name = Busty::TOPLESS_SIMON_MAP[$game_message.face_name]
+      ]
     end
```

Comment:
Can be a simple if, as there's an early return before.

Also, I think we can do without the map by putting the topless images in a `topless/` subfolder and keeping them named identically otherwise:
```rb
if $game_actors[2].character_index == 2
  return [
    -40,
    19,
    "topless/" + $game_message.face_name,
  ]
end
```

Reply:
We only want to trigger this if it's Simon being displayed, so it's not sufficient to only check the character index setting. We also only want to display it for facesets that have a corresponding topless Simon; there's no topless Megabeard Simon, for instance. I'd also prefer to stick to Sierra's convention for having everything in the Faces/ folder with different filenames, rather than using duplicate filenames in subfolders.

Reply:
Yes, I made it work in all cases where Simon is naked, so it's applicable to Simon2 as well as Simon1. I didn't bother with megabeard or wedding Simon because they'd never be naked.

I deliberately named the files that way so they'd be grouped together with the corresponding clothed files, so that Sierra can easily find them to use in events if she so chooses.

Reply:
> I deliberately named the files that way so they'd be grouped together with the corresponding clothed files, so that Sierra can easily find them to use in events if she so chooses.

Please double check with Sierra that you won't be messing with her muscle memory that way. Renaming files is easy while the PR isn't merged but will be a mess after the files are copied in a folder with no version history on Sierra's computer.

I know, I know, probably overly cautious there. But the prod is currently crashing (the gradient memory leak thing) because of too much "going fast" lately so...

Reply:
Fair point, will do.

Reply:
Sierra's response:

> These are the files for unclothed characters like Simon? If these are facesets I might hypothetically use in the future, they should be beside the other facesets.